integration_tests/volume_example.py

```diff
@@ -0,0 +1,59 @@
+import os
```

Comment:
This is not executed on CI yet. You need to add it to https://github.com/couler-proj/couler/blob/master/scripts/integration_tests.sh.